src/frontend/src/styles/main.css

```diff
@@ -575,6 +575,11 @@ a:hover,
   padding: var(--rs-card-bezel-narrow);
 }
 
+.c-card--tight {
```

Comment:
`tight` is like `narrow` but takes up more horizontal space, is that it?

Reply:
tight is even tighter than narrow but when the space is too small it looks unequal in horizontally and vertically because of the lineheight. So we correct that with some uneven spacing